src/Features/LanguageServer/Protocol/Extensions/ProtocolConversions.cs

```diff
@@ -65,7 +65,7 @@ internal static class ProtocolConversions
         /// Workaround for razor file paths being provided with a preceding slash on windows.
         /// Long term fix in razor here - https://github.com/dotnet/aspnetcore/issues/19948
         /// </summary>
-        public static Uri GetUriFromFilePath(string filePath)
+        public static Uri GetUriFromFilePath(string? filePath)
```

Comment:
It's going to throw if this is passed in null, so this annotation doesn't seem correct.